### PR TITLE
Add changelog for 14.7.0

### PR DESCRIPTION
**Pull request by elcct**
### Checklist

* [ ] Has a new newsfragment been created? Check [changelog/README.rst](https://github.com/uktrade/data-hub-leeloo/blob/master/changelog/README.rst) for instructions
* [ ] Do any added or updated endpoints appear in the API documentation? See [docs/Maintaining the API documentation.md](https://github.com/uktrade/data-hub-leeloo/blob/develop/docs/Maintaining&#32;the&#32;API&#32;documentation.md) for more details
* [ ] Have any relevant search models been updated?
* [ ] Have any relevant fixtures (`fixtures/test_data.yaml`) been updated?
* [ ] Have any relevant select-/prefetch-related field lists in the views and search apps been updated?
* [ ] Has the admin site been updated (for new models, fields etc.)?
* [ ] Has the README been updated (if needed)?

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,3 +1,95 @@
...
+  By default, the Celery task runs asynchronously. ``--foreground`` can be passed to run the Celery task synchronously (without Celery running).
+
+  The ``--batch_size`` argument was removed as it is rarely used and isn't currently supported by the Celery task.
+
+API
+---
+
+- **Advisers** The following endpoint was added:
+
+    - ``PUT /v4/company-list/<company list ID>/item/<company ID>``
```
Comment (reupen) [inline CHANGELOG.rst]: This one is rendering as a quote, is that intended?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,3 +1,95 @@
...
+  ``latest_interaction`` may be ``null`` if the company has no interactions.
+
+  Results are sorted by ``latest_interaction.date`` in reverse chronological order, with ``null`` values last.
+
+  The endpoint has pagination in line with other endpoints; to retrieve all results pass a large value for the ``limit`` query parameter (e.g. ``?limit=1000``).
+
+  If selected list does not exist, the endpoint will return 404 status code.
+- **Advisers** The following endpoint was added:
+
+    - ``DELETE /v4/company-list/<company list ID>/item/<company ID>``
```
Comment (reupen) [inline CHANGELOG.rst]: This one is also rendering as a quote.

File: CHANGELOG.rst (hunk 1 above)
Comment (elcct) [reply CHANGELOG.rst]: Nice! Didn't spot that. Yes that's wrong and I just corrected it. Thank you!

File: CHANGELOG.rst (hunk 2 above)
Comment (elcct) [reply CHANGELOG.rst]: As above. Thank you!

**Pull request merged**